docs/source/conf.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yup. They're just to preserve the `/_static/` directory, which we don't need, but the build script will give you warnings if it doesn't exist.